browse-scaler/src/index.js

```diff
@@ -67,10 +67,16 @@ const resizeImageFromConceptId = async (conceptType, conceptId, height, width) =
 
   // If given an image url, fetch the image and resize. If no image
   // exists, return the not found response
-  const imageUrl = await getImageUrlFromConcept(conceptId, conceptType);
-  const imageBuffer = await withTimeout(timeoutInterval, slurpImageIntoBuffer(imageUrl));
+  const imageUrl = await withTimeout(timeoutInterval, getImageUrlFromConcept(conceptId, conceptType));
+  if (imageUrl === null) {
+    console.log("IMAGE URL IS NOT RETURNED.");
```

Comment:
I think we should either remove this log statement or add the concept id to make this log message more useful. We can just use string templating like so: ```console.log(`No image url found for concept: ${conceptId]`);```